### PR TITLE
[Telink] Add OTA LZMA image compression & Update builds to docker version 90

### PR DESCRIPTION
**Pull request by s07641069**
**Change overview**

- Add OTA LZMA image compression
- Add manual OTA test by button
- Update compatible builds to docker version 90 https://github.com/project-chip/connectedhomeip/pull/36355

### CODE REVIEW

**Comment by s07641069** [comment]
Require new Telink Docker image https://github.com/project-chip/connectedhomeip/pull/36355

**Comment by s07641069** [comment]
@andy31415 
Somehow reviewers are not assigned automatically in latest PRs

**Comment by andy31415** [comment]
> @andy31415 Somehow reviewers are not assigned automatically in latest PRs

@s07641069 this change was intentional as a result of discussions in the software development group: adding every possible reviewer to every PR did not seem to be productive: most people would not end up reviewing anyway and it seemed like mostly noise. As a result, we stopped adding people to PRs and rely on previous auto-assignment (that was done anyway when the entire list was added) or by following up on slack.

The only auto-add is done for area-specific settings defined in https://github.com/project-chip/connectedhomeip/blob/master/CODEOWNERS

**Review by andy31415 — APPROVED** [review]

File: scripts/build/build/targets.py [hunk 1]
```diff
@@ -800,6 +800,7 @@ def BuildTelinkTarget():
     target.AppendModifier('4mb', enable_4mb_flash=True)
     target.AppendModifier('mars', mars_board_config=True)
     target.AppendModifier('usb', usb_board_config=True)
+    target.AppendModifier('compress_lzma', compress_lzma_config=True)
```
Comment (arkq) [inline scripts/build/build/targets.py:803]: The convention is that everything is separated with "-", so it's easier to type it.

nit:
```suggestion
    target.AppendModifier('compress-lzma', compress_lzma_config=True)
```

**Review by arkq — APPROVED** [review]

**Pull request merged**